code/game/objects/items/weapons/tools.dm

```diff
@@ -358,7 +358,7 @@
 	var/safety = user:eyecheck()
 	if(ishuman(user))
 		var/mob/living/carbon/human/H = user
-		var/obj/item/organ/eyes/E = H.internal_organs_by_name["eyes"]
+		var/obj/item/organ/internal/eyes/E = H.get_int_organ(/obj/item/organ/internal/eyes)

```

Comment:
Well, looking at things further, things will work just fine as-is, without repathing IPC optics. If we think it's necessary, we can look into trying something like that out later.